src/svc_vc.c

```diff
@@ -151,12 +151,6 @@ svc_vc_ncreate2(int fd, u_int sendsize, u_int recvsize, u_int flags)
 	/* attach shared state */
 	if ((oflags & RPC_DPLX_LKP_OFLAG_ALLOC) || (!rec->hdl.xd)) {
 		xd = rec->hdl.xd = alloc_x_vc_data();
```

Comment:
Didn't even know we had a tirpc_malloc()....  I'll look into it for a different patch.